### DISCUSSION TITLE
Github Pages with a private repo

### DISCUSSION THREAD

**Discussion by ArabiaGS** [community/community#102018] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

I am developing a project that involves a html website. I looked into Github Pages, but I don't want to expose the repository, just the website. Would it be possible to make a public page from a private repository with Github Pro?

Thanks

**Comment by shubhangamjha** [comment]
yeah definetly. Just give access to the deployment sites and its done.

**Comment by B0K1NG** [comment]
Yes it is possible, just check the documentation on how to deploy them: https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site

**Reply by ArabiaGS** [reply]
I was looking into that and that's where I got confused, because when I enter on Github Pages with a private repo, the upgrade option is pro, but that document talks about enterprise cloud

**Comment by sbk2k1** [comment]
Hey @ArabiaGS you can also look at this blog to setup GitHub actions to deploy your site to GitHub Pages using a custom action. It deploys an updated website when you commit to the repo.

https://highonbugs.sbk2k1.tech/how-to-deploy-your-website-to-github-pages-using-github-actions

It works just fine on private repos as well.
